src/routes/api/orderStatus.routes.js

```diff
@@ -1,9 +1,14 @@
 import { Router } from 'express';
 import extractToken from '../../middlewares/checkUserWithToken';
-import trackOrderStatus from '../../controllers/orderStatus.controller';
+import {
+  trackOrderStatus,
+  getOrders,
+} from '../../controllers/orderStatus.controller';
 
 const route = Router();
 
+route.get('/all', extractToken, getOrders);
```

Comment:
Are all users (buyer,seller,admin) able to track orders??
And you should check if user password is not expires

Reply:
@ba-lambert I have updated that route, and I think password expiration is checked when user is gonna login.

Reply:
Also you should check the role of a user